05-cleaning-data/01-intro.md

```diff
@@ -0,0 +1,15 @@
+# Cleaning Data
+
+![Data Science Process](https://docs.google.com/presentation/d/114QYFmKuJ2M5E3tlBw8Gwu2xI09tb8gnrKrNkMY9CG4/export/png?pageid=g145cde68be4_0_70)
+
+## Learning Objectives
+
+You have already learned about what tidy data is, why it is important, and how to tidy up a dataset in the main Data Trail course (see [here](https://datatrail-jhu.github.io/DataTrail/cleaning-data.html) for a refresher). After completing this section, you will be able to:
```

Comment:
This is great! However it won't render properly if it isn't listed in the bookdown.yml and bookdown doesn't do well with subdirectories. Instead you can make separate chapters in the same 05-cleaning-data.Rmd file by creating a new H1 header. 

We can chat about this later today!